### PR TITLE
battle_anim_effects_2 labels

### PR DESCRIPTION
**Pull request by Sewef**

### CODE REVIEW

**Comment by Sewef** [comment]
Ready for review

File: src/battle_anim_effects_2.c [hunk 1]
```diff
@@ -680,130 +686,130 @@ const union AffineAnimCmd gUnknown_08593764[] =
...
 
-const union AffineAnimCmd *const gUnknown_0859384C[] =
+const union AffineAnimCmd *const gThingRingShrinkingAffineAnimTable[] =
 {
-    gUnknown_08593834,
+    gThingRingShrinkingAffineAnimCmds,
 };
 
-const struct SpriteTemplate gUnknown_08593850 =
+const struct SpriteTemplate gThingRingShrinkingSpriteTemplate =
```
Comment (huderlem) [inline src/battle_anim_effects_2.c]: typo: `gThing` instead when you meant `gThin`

Comment (huderlem) [reply src/battle_anim_effects_2.c]: There are other examples of this typo besides this line.

**Comment by Sewef** [comment]
Updated

**Pull request merged**